plone/app/discussion/conversation.py

```diff
@@ -90,11 +90,19 @@ def enabled(self):
     @property
     def total_comments(self):
         public_comments = [
-            x for x in self._comments.values()
+            x for x in self.values()
             if user_nobody.has_permission('View', x)
         ]
         return len(public_comments)
 
+    def another_total_comments(self):
```

Comment:
Wow, I would have never expected that, although obvious in this case I didn't though it would be a general issue.

So should we create a p.r.codenalaysis check that recommends migrating all ``@property`` to ``ComputedAttribute``? Is ``ComputedAttribute`` a drop-in replacement?